### DISCUSSION TITLE
Light Source Calibration Issue

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#29] [Q&A]
Hi Serhiy,

So I started with a calibration of 12.5v as recommended from Light Source Wiki.   I don't have a fancy luminance meter but I do have various camera hot shoe meters and digital cameras.   At 14.03 K value, EV12 with a voltage set to 12.95v, I am reading light output a full -1.7 to -2 stop below what it should be.  

This is after swapping RCS to 0.091 Ohm.

I've got 12.95v from power supply and 12.95v at LED board's 12V terminals.    At full brightness there is 10.9v at the "LED" terminal.  

The only way I can get it to match is if I raise voltage above 13v (I think it was maybe around 13.5v cant remember).  Your instructions clearly sate this can damage components, so I quickly brought it back down to 12.9v, but output is not enough.

Any suggestions?

**Comment by srozum** [comment]
Well, first I assume you have correct LED, XHP7070.2 (not Cree XHP 70.2), and fresnel lens is placed correctly.
Not clear what diffuser you've got for your light unit.

With correct LED and correct Diffuser light unit should output 17ish (about 17.2EV, measured with modern digital meter) when fully ON. "Fully ON" is when tester in time measurement mode like Simple or Focal Shutter, regardless of what K is set in settings.

I hope you understand part about +1EV shift. If you are using digital camera for calibration, settings should be set to K14.03 (most modern cameras calibrated to 14.03) and EV shift to 0EV (no shift).

Maybe add a photo of your diffuser.

And don't exceed 13V, you may burn the fan motor.

**Comment by polygonfuture** [comment]
Calibration tests were done using the LIGHT mode.    K14.03 set, EV set to 0.   Light mode set to EV12  (which reads ISO 100, F8, 1/60).

**LED is this:**
![image](https://user-images.githubusercontent.com/12026545/231681904-e4fd347b-7e66-4771-9d0b-0dd8bfd9ff06.png)
[https://www.aliexpress.us/item/2251832705995442.html?spm=a2g0o.order_list.order_list_main.86.21ef1802JotJK3&gatewayAdapt=glo2usa&_randl_shipto=US](https://www.aliexpress.us/item/2251832705995442.html?spm=a2g0o.order_list.order_list_main.86.21ef1802JotJK3&gatewayAdapt=glo2usa&_randl_shipto=US)

**Diffuser material:**
[https://www.tapplastics.com/product/plastics/cut_to_size_plastic/satinice_white/655](https://www.tapplastics.com/product/plastics/cut_to_size_plastic/satinice_white/655)
Says 59% light transmission.   Its matte on both sides.

I will attach a photo in the morning when I have better light of the diffuser material.

Fresnel lens is placed in its slot in the LED mount assembly.  This is sandwiched between the larger cone and the led mount.   It did not seem like it mattered which side faced outward with tests I did.  I believe I did smooth side facing outward, but would have to disassemble and check.

**Reply by srozum** [reply]
Ok, good thing you have access to Tap Plastics. Now go back to them and ask for "Silky White" 6mm thick, glossy on one side (facing inwards), matte on other side (facing outwards). I think it this one https://www.tapplastics.com/product/plastics/cut_to_size_plastic/acrylic_sheets_p95_matte/513
In store they different names. If you can walk into their store you may get free sample big enough for the light unit.

About LED I specifically said in BOM: Chinese version called 7070.2, not Cree 70.2. 70.2 rated for higher current, means it less bright at 12.5V and current provided by driver. That's why you should use Chinese LED.

Orientation of fresnel lens does matter. It provides evenness of illuminated surface. Smoot side towards LED, with groves facing forward.

**Reply by polygonfuture** [reply]
The light was my mistake.  I thought it was 7070.2 not 70.2.   Aliexpress sellers label most Epileds also as Cree as well in the name and it makes it very confusing.   Also many labeled as 7070.2 still show CREE on their images, etc.

So this is the correct LED then?   If I replace current with this then output should be brighter is what you are saying right?
[https://www.aliexpress.us/item/3256802823007929.html?spm=a2g0o.order_list.order_list_main.5.5d1c1802Xp0yht&gatewayAdapt=glo2usa&_randl_shipto=US](https://www.aliexpress.us/item/3256802823007929.html?spm=a2g0o.order_list.order_list_main.5.5d1c1802Xp0yht&gatewayAdapt=glo2usa&_randl_shipto=US)

Thank you for clarifying fresnel direction.  I will double check and flip if necessary.  It appears even to my eyes but maybe that is because of the specific diffuser material I'm using.

Also, there are currently no directions for assembling the calibrated light unit.   This fresnel information would be good to have posted so that others do not make the same mistake.

**Comment by srozum** [comment]
> Also, there are currently no directions for assembling the calibrated light unit. This fresnel information would be good to have posted so that others do not make the same mistake.

I'll leave it as is. People who work with cameras usually know which way condenser lens should be oriented. :-)

**Reply by polygonfuture** [reply]
Hi @srozum 

So I purchased a new set of XHP 7070.2 and of course it arrives and the LED once again says XHP70.  

Can you recommend a vendor that we can find 7070.2 leds from?   It seems everything I'm seeing are cree analogs even if they say they are 7070.2 in the description.  Its a crap shoot.

**Comment by srozum** [comment]
Copper board may say XHP70, but XHP7070 is more yellowish and have more distinct lines between four segments of a chip.

![IMG_2693](https://user-images.githubusercontent.com/56254/233764642-c29de437-5542-4658-a93b-35285269a85c.jpeg)

**Reply by polygonfuture** [reply]
Can you upload this image at higher res?  Cant see emitter very clearly on my screen.

**Reply by polygonfuture** [reply]
Thanks Serhiy.  Ill take a look tomorrow and compare.  From what I tell latest (which was said to be 7070.2) is missing two dots on the led die i see, and lines are not as distinct.   Silicone is clear but LED chip is very matte looking on mine. Yellow for sure but hard to tell compared to other.  WIll share photos tomorrow or next day.

**Reply by polygonfuture** [reply]
Posting an update.  Here's latest LED's branded as EPILEDS from Aliexpress sellers.   I've purchased two different orders from two suppliers on aliexpress.   

RZGTOP LEDlight Store
LED World company

The second supplier "LED world company"  specifically listed as EPILEDS and was the only to do so in the product title.  RZGTOP listed epileds in product description only.   Also for the uninitiated, pro tip is you cannot trust product images on aliexpress.  Always ask seller before purchasing.    LEDS from both suppliers look the same but still need to test output. 

Here is what my original LED chip (cree version with wrong light output) looked like, and the latest one from LED World company seller:

![cree_vs_epileds_versions](https://user-images.githubusercontent.com/12026545/236314652-93472c19-de2e-4ec2-bc1f-da963b9d7290.png)

**Comment by srozum** [comment]
So what the result? If Light Unit produces LINEAR output then you got right LED.

Also, did you get right diffuser?

**Reply by polygonfuture** [reply]
Just tested and Light source is linear according to a Sony A7SII with 50mm lens, but still outputting below expected EV by -1.7 f stops.
My assumption is because of the diffuser material.   New material should arrive next day or two and will test again once arrives.  

Also for any U.S. builders if you order form TAP plastics there is no material called "Silky White".   Though as Serhiy has mentioned in previous threads P95 matte frosted finish should be that.   Will confirm once received.

**Reply by polygonfuture** [reply]
**Updates:**     Diffuser made zero difference in readings.  

I made some adjustments to how I wired LED to PSU and that improved light output closer, but not great.  Light output is still  -0.7 to -0.3 stops less than it should be.  1/1000 and from 1/4 and below, light output measure around -0.3 stop below.   Everything in between is -0.7 stop slower.  Voltage output from MeanWell 25-12 is set to 12.9v.

LED Board power is running direct from PSU to 12V input on board.  Wire is 22AWG stranded silicone wire.  Should this be larger AWG?   

I'm not sure why its still almost a full stop below at such a high voltage.   I'll try an LED from RZGTOP that ic-racer used next I guess.  
Beyond that any thoughts or suggestions @srozum?

**Reply by polygonfuture** [reply]
Update 2:  

LED was correct.  Swapped it for confirmed RGZTOP supplier that IC-Racer used and got the exact same results.  Double check linearity and it is consistently -0.7 stops below expected output.

I guess next step is to increase wire gauge to 18/16ga  from PSU to LED and hope that solve the issue. 
After that I can try replacing the LED driver with a new one.

**Comment by srozum** [comment]
Wire doesn't play such a big role. You mentioned before that "At full brightness there is 10.9v at the "LED" terminal." I'll suggest you double check your driver and RCS. If you have spare driver, take measurements before swapping RCS and after, to have better idea.

LED driver is a CURRENT regulator, so voltage on LED should be equal to voltage on power terminal minus some mili volts drop caused by a driver. But...

> I hope you understand part about +1EV shift. If you are using digital camera for calibration, settings should be set to K14.03 (most modern cameras calibrated to 14.03) and EV shift to 0EV (no shift).

Please, describe how do you measure.

**Reply by polygonfuture** [reply]
LED Driver does not appear to be a current regulator.   As brightness changes voltage is also changing.    I bought exact model from datasheet so seller must be selling device that is not correct.

Tested and I'm getting 11v at full brightness EV16 of LED.     
RCS is using 091 resistor exactly as described, and from a reputable U.S. source.

**Reply by polygonfuture** [reply]
FYI, LD24AJTA boards are not all current regulating.   Many claims to be constant current.   Not sure what's going on there, but both of mine appear to be changing voltage not current.    Maybe both are defective I dont know.

**Reply by srozum** [reply]
Could you please answer my question: describe how do you measure. LU settings, Light Meter settings/parameters, how setup looks like.

**Reply by polygonfuture** [reply]
Manual mode, Sony A7S with 50mm lens set to infinity.

f8, iso 100, manual exposure.    Camera lens placed right up to front of EE tester.   Metering is center mode but changing has no real effect on exposure.  

LU is set as before 14k with no exposure compensations  set.  LU is put into light test mode and each EV is cycled through.  I set camera to each shutter speed setting and record f-stop difference.   Light is Consistently 0.7 stop below expected value. 

I will test your 1ev under voltage experiment tomorrow and post results.

**Reply by srozum** [reply]
Thanks. Also would help some photos of you unit, driver, diffuser. It's hard to troubleshoot like that.

**Reply by polygonfuture** [reply]
Here’s some photos for you. 

![IMG_8570](https://user-images.githubusercontent.com/12026545/236730882-ea6b080c-b081-4f6c-aec2-a88f4a7a15cc.jpeg)
![IMG_8569](https://user-images.githubusercontent.com/12026545/236730890-166c58de-c9d6-4cb7-a61a-23a38eb89ecc.jpeg)
![IMG_8574](https://user-images.githubusercontent.com/12026545/236730897-f398786a-2af1-4fa3-9a9e-1cba01b17d36.jpeg)
![IMG_8575](https://user-images.githubusercontent.com/12026545/236730996-6bd579f6-bbf0-4b11-8946-a7ecd0945550.jpeg)

**Reply by srozum** [reply]
I see that diffuser looks like a right one. PCB and driver correct. Assume that fresnel lens installed correctly. LED you already tested few versions. Everything looks fine to me.

Problem must be in something else. Either driver. As I said, you might want to compare unmodified one with modified and see if RCS makes a difference. Either you measuring wrong. Check using other light meters or cameras. -0.7..-0.3EV is much closer than used to be. So keep digging.

Nice build!

**Comment by srozum** [comment]
Do some experiment - whatever you using for calibration what tells you that LU is 0.7-0.3EV under, calibrate LU to be exactly 1EV under, and tell me at what voltage you get.

**Comment by ic-racer** [comment]
"Digital Camera" as light meter? My D90 does not read the same as a film camera meter. 

 **ISO 12232:2019 Digital still cameras — Determination of exposure index, ISO speed ratings**
_"This document was designed to harmonize with earlier standards developed for film-based photography...However, there are differences between electronic and film-based imaging systems that preclude exact equivalency. "_

**Comment by polygonfuture** [comment]
@srozum,

I tested the first LED board I soldered with the LED driver that had already been swapped RCS 091.  I then tested with a new board RCS to 091.   

I'm getting the same values for both.   

I adjusted until EV is -1 under using the A7SII with 50mm (infinity) and at f8 and these are the voltages I get at each EV range:

![image](https://github.com/srozum/film_camera_tester/assets/12026545/c9f9fbfd-feb9-4873-95a5-f14e98fced8b)



Also for sanity check, I was wondering if this luminance meter by UNI-T might be accurate enough to test light source to rule out any issues possibly coming from my camera?

**https://www.amazon.com/UNI-T-Illuminance-Photography-Photometer-Measurement/dp/B09PDGX69B/ref=asc_df_B09PDGX69B/?tag=hyprod-20&linkCode=df0&hvadid=647264383369&hvpos=&hvnetw=g&hvrand=16366537249684943398&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1013962&hvtargid=pla-1807988982469&psc=1&gclid=CjwKCAjw4ZWkBhA4EiwAVJXwqV76F_uL8gl-bK6x_Z0da0LiOyLrfopIo2_9-Jt6FAIcLmj2YVjUHhoCZVMQAvD_BwE**

**Reply by polygonfuture** [reply]
I realized I have an x-rite monitor display calibrator  i1 Display Pro (from like 2013-2015 maybe).    I remember it had a luminance measure function on it.

Im not sure if they are doing anything special under the hood for how they calculate luminance, but just to give a secondary source of measurement:

I placed the sensor onto the light source at EV12 using K12.3 per "Light Source" calibration document and I get measurement of 346.5 cd/m2.

**Reply by srozum** [reply]
Good that you have actual Luminance meter. If I understand correctly, at 12.24V power supply you measured 346.5 cd/m2 at EV12 K12.5. Look at calibration table here https://github.com/srozum/film_camera_tester/wiki/Light-Source#light-unit-2 

It seems that you have to increase voltage a big to get 512 cd/m2 at EV12 K12.5. Then measure other EV, confirm that those are all close to number in a table and you are good.

Make sure that you place you x-rite at the center of the screen and that it always the same if you have to remove it for adjustments.

**Reply by polygonfuture** [reply]
The only way I can reach 512 cd/m2 at EV12 is when voltage is at 13.8V.

At this point, "LED" labeled terminal reads voltage of 10.5V and "FAN" terminals read at 13.8V as well, which as you say will likely fry fan motor.

Still seems something is wrong.

**Reply by polygonfuture** [reply]
I checked through the entire range.  EV5 - EV13 match close +/- 5 numbers.   Above 13 though and I believe this illuminance tester begins to no longer function properly.  EV14 will read 2135, EV15 as 2500, and EV16 as 2350.  

Again this is K value of 12.5, running at 13.8V from PSU.

**Reply by srozum** [reply]
Yeah, it shouldn't take more than 1V to change LED brightness from 346 up to 512 candelas. Definitely either wrong LED or x-rite not a correct tool for this.

I've used "SM208 Screen Brightness Meter Screen Luminance Meter" https://www.amazon.com/SM208-Screen-Brightness-Meter-Luminance/dp/B00H050VEI

I am actually out of ideas. If you want to finish your tester I might have a solution. As I understand you are somewhere in LA area. What if you ship your Light Unit to me in SF, I'll finish it and ship it back to you. I have spare LED and Diffuser. Find me on FB and shoot a message if it looks like a good idea.

**Reply by polygonfuture** [reply]
This sounds like a good plan.  I am going to do test today by hooking up LED driver board directly my bench supply and see if somehow the Meanwell PSU is not able to supply enough current?  

That is the only thing other than something still wrong about LED that I can think of.    Let me test supply today and see if that makes any difference.  Otherwise I will reach out on FB about sending to you.   Thank you so much for all this support Serhiy.

**Reply by polygonfuture** [reply]
So just did a test with bench supply and it matches tests with Meanweell PSU.

At 13.90v with 0.45A current draw from bench supply I get 512 cd/m2.    Looks like I'll be reaching out on FB.

**Comment by srozum** [comment]
Issue was with a fresnel lens of wrong focal length. Closing
